web/app/js/components/Sidebar.jsx

```diff
@@ -148,54 +204,31 @@ class Sidebar extends React.Component {
               </PrefixedLink>
             </Menu.Item>
 
-            <Menu.Item className="sidebar-menu-item" key="/namespaces">
```

Comment:
we still want to access the list pages (/namespaces, /deployments, /pods) etc. from somewhere so let's make sure these don't get lost